app/slackimport.go

```diff
@@ -164,7 +161,8 @@ func (a *App) SlackAddUsers(teamId string, slackusers []SlackUser, importerLog *
 		password := model.NewId()
 
 		// Check for email conflict and use existing user if found
-		if result := <-a.Srv.Store.User().GetByEmail(email); result.Err == nil {
+		result = <-a.Srv.Store.User().GetByEmail(email)
```

Comment:
No need to split this one out, as the `result` variable isn't needed anywhere other than the `if` block, and there isn't any `else`.

Reply:
Makes perfect sense, thank you!